### PR TITLE
Add statmorph recipe.

### PR DESCRIPTION
**Pull request by vrodgom**

### CODE REVIEW

File: recipes/statmorph/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
+
```
Comment (dougalsutherland) [inline recipes/statmorph/meta.yaml]: Nit: you could drop this blank line.

File: recipes/statmorph/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - scikit-image >=0.13
```
Comment (dougalsutherland) [inline recipes/statmorph/meta.yaml]: You should replace the `build` requirements with just `python` and `setuptools.` The explicit `python` dep gets used by conda for various things, and once you're explicit about `setuptools`, you won't actually need `scikit-image`/`astropy`/`photutils` at build time, just for `run`.

File: recipes/statmorph/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    - astropy >=2.0
+    - photutils >=0.3
+  run:
```
Comment (dougalsutherland) [inline recipes/statmorph/meta.yaml]: Add an explicit `python` requirement to `run`, please. `conda` uses it sometimes.

File: recipes/statmorph/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+  imports:
+    - statmorph
+
+about:
+  home: https://github.com/vrodgom/statmorph
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE
+  summary: 'Non-parametric morphological diagnostics of galaxy images'
+
```
Comment (dougalsutherland) [inline recipes/statmorph/meta.yaml]: Nit: you could drop this blank line.

File: recipes/statmorph/meta.yaml (hunk 1 above)
Comment (vrodgom) [reply recipes/statmorph/meta.yaml]: Thanks! This should be taken care of.

Comment (jakirkham) [reply recipes/statmorph/meta.yaml]: It is. Just GitHub diffs can be dumb sometimes and hence move comments to match the recent PR diff. 😞

**Comment by jakirkham** [comment]
Thanks @vrodgom. Also thanks @dougalsutherland for reviewing.

**Pull request merged**